python/marvin/tools/map.py

```diff
@@ -386,3 +417,257 @@ def plot(self, array='value', xlim=None, ylim=None, zlim=None,
             return (ax, fig)
         else:
             return ax
+
+    def _make_image(self, snr_min, log_cb, use_mask):
+        """Make masked array of image.
+
+        Args:
```

Comment:
Especially since some of our methods use (args, and kwargs).  It may give the impression to users we are only documenting the possible args and not kwargs.